### ISSUE TITLE
sub care plans

### ISSUE THREAD

**Issue by joostholslag** [SanteonNL/shared-care-planning#35]
In my experience as a doctor and clinical modeller the care team depends on the topic of the care plan. So different topics lead to different care teams. e.g. a nurse can be involved in the diabetes care plan, but not in the depression care plan. So a care team membership because of involvement in diabetes care shall not lead to an authorisation for the depression related data. https://github.com/SanteonNL/shared-care-planning/blob/main/input/pagecontent/introduction.md implies there will be only one care team per patient.
My solution was to model sub care plans ([plan elements](https://archetype-editor.nedap.healthcare/advanced/plan/archetypes/openEHR-EHR-COMPOSITION.plan_element-SFMPC.v1.0.0)) as self standing artefacts, and have an ['overarching/main' care plan](https://archetype-editor.nedap.healthcare/advanced/plan/archetypes/openEHR-EHR-COMPOSITION.plan-SFMPC.v0.3.0/opt) that indexes the relevant sub care plans for a given audience.

**Comment by bramwesselo** [comment]
Yeah, we more or less also ended up at your solution; the CarePlan as mentioned in SCP is an overarching 'network-careplan'; it will not contain every action for the patient as multiple organizations might have their own careplan (locally known as e.g. episode, DBC or any other condition/treatment related container-object) where actions are registered. Ideally, these local careplans are referring to the SCP-careplan.

Still, the SCP-Careplan should be some conditions. If diabetes and depression are related conditions, the SCP-careplan should refer both.

**Comment by joostholslag** [comment]
To me the main point is the care team may differ per condition. So if you want to deduce authorisations from a care plan with a care team, you must have a care plan with care team per condition.
